src/main/java/com/dpgil/pathlinker/path_linker/internal/Algorithms.java

```diff
@@ -143,13 +143,16 @@ public int hashCode()
      *            the target node
      * @param maxK
      *            the number of shortest paths
+     * @param includePathScoreTies
+     * 			  the option to include all paths of equal length
```

Comment:
This option does not include all paths of equal length. Rather, when the maxK path is reached, all paths with the same weight are included